edi_webservice_oca/security/ir.model.access.csv

```diff
@@ -1,2 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_webservice_backend_edi_manager,webservice_backend edi manager,webservice.model_webservice_backend,base_edi.group_edi_manager,1,1,1,1
+access_webservice_backend_user,webservice_backend user,webservice.model_webservice_backend,base.group_user,1,0,0,0
```

Comment:
I've realized that this would gives read access to credential, it's probably better to use sudo insteads